### ISSUE TITLE
No Streams Available???

### ISSUE THREAD

**Issue by nicole80x** [mrknow/filmkodi#892]
Every time I try to watch something it says there are no streams available. Please help me with this issue.

## Please follow the guide below 
- Put an "x" into all boxes [ ] relevant to your *issue*
- Use *Preview* tab to see how your issue will actually look like
---
## Kodi & Addon 
- Kodi verison: _____
- [X ] Filmy online - Mrknow
- [ X] MrknowTV
- [ ] Specto
- [ ] FanFILM
- Version / Wersja: _____
---
## Descryption / Opis
Put here every imported thing such logs, how to reproduce etc. <br/>
Wstaw tutaj każdą ważna rzecz, taką jak logi, jak wywołać błąd itp.

## Donation / Darowizna
This addons are free, but you can always thank the author for ... / Te wtyczki są darmowe, ale zawsze możesz podziękować autorowi za jej stworzenie...

[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=HEV6TJY83S3Q4&lc=PL&item_name=Donation-Darowizna&currency_code=PLN&bn=PP%2dDonationsBF%3abtn_donateCC_LG%2egif%3aNonHosted)

**Comment by JTVesagie** [comment]
hi mate , i have the exact same issue , was a solution ever found  . im currently experiencing it on the crew and home lander

**Comment by BankruptEd** [comment]
I haven't used Kodi in ages bro. Good luck though.

**Comment by JTVesagie** [comment]
Thank you . i actually did come right believe it or not . this kodi is very
frustrating . but at least working for now . But i definitely know it much
better now after trying to resolve this issue

**Comment by BankruptEd** [comment]
Absolutely. The more you mess with it, the better you'll get. I just got
tired of all the work to keep everything going and switched to other apps
that were less prone to give me a headache lol.

**Comment by JTVesagie** [comment]
im very close to that point , but also I think that instability of kodi is
almost addictive , i cant just admit defeat . It really will have to jerk
me around badly before i decide im wasting too much time here . But i think
it really might be worthwhile at some point to get debrid . only issue i
have with that is the method of payment , or atleast i wasnt happy with it
when I last checked . But you have done the right thing , there comes a
time when you are fed up with tinkering around everytime 2 nd time you want
to watch something

**Comment by BankruptEd** [comment]
Exactly.  But I get what you're saying about Kodi. It's kind of a hobby as
well as a way to get free TV. I use Real Debrid with Cinema HD. I don't
think Cinema HD is supported anymore but it's still working good for now.

**Comment by JTVesagie** [comment]
A hobby , probably the perfect way to put it , im guessing what you
currently using is almost problem free . i know im going to run into
difficulties again , ill have a look at that option . maybe its trading
under some other name or something . rIGHT NOW IM KING OF THE WORLD HAVING
RESOLVED THE PROBLEM , BUT ITS MOMENTs AWAY before im brought back down to
earth . All depending on how much time I have or if I really want to go
down that fix kodi hole  will determine if i move on then or the next time
.lol

**Comment by BankruptEd** [comment]
lol I hear ya bro. Ya my setup has been trouble free for several years now.
I started out with apps where you didn't even need Real Debrid, but those
days are gone and been using RD for past few years and have no complaints.
Good luck with the Kodi my friend  It'll always have a place in my heart as
it was my introduction to free TV. ✌️

**Comment by JTVesagie** [comment]
lol thanks , already run into a new problem , think ill also be looking
back on kodi with fondness I hope . ill give this issue a couple of goes
but life is to short to continuously keep smacking your head into a wall .
but good luck , it woud be really nice if the kodi could be refined a bit

**Comment by BankruptEd** [comment]
Yes for sure. A little streamlining is in order lol.
